Products/ZenMessaging/ChangeEvents/events.py

```diff
@@ -63,6 +63,8 @@ class MessagePrePublishingEvent(object):
 
     def __init__(self, msgs, maintWindowChanges):
         self.msgs = msgs
+        self.refs = refs
```

Comment:
refs does not seem to be a parameter in __init__

Reply:
Thank you!.. updated